File: Python/1180 - Lowest Number and Position.py
if __name__ == "__main__":
    useless = int(input())
    l1 = input().split()
    for i in range(len(l1)):
        l1[i] = int(l1[i])
    min_val = min(l1)
    min_ind = l1.index(min_val)
    print("Menor valor: %d\nPosicao: %d" % (min_val,min_ind))

File: C++/1180 - Lowest Number and Position.cpp
#include <iostream>
#include <vector>

int main()
{
    std::vector<int> vect;
    int lowest_pos, low,n,x;
    std::cin >> n;
    for(int i = 0; i < n; i++)
    {
        std::cin >> x;
        vect.push_back(x);
        if(i == 0 ||low > x)
        {
            low = x;
            lowest_pos = i;
        }
    }
    printf("Menor valor: %d\nPosicao: %d\n",low,lowest_pos);
    return 0;
}